### DISCUSSION TITLE
How can I use bud to make a login page?

### DISCUSSION THREAD

**Discussion by d3vw** [livebud/bud#286] [Q&A]
Hey guys!
Through examples, I have learned that bud can render json data.
But how can I use this to make a user login page.
just like this:

Username: grey
Password: 1234

And then check to see if it's legal
and login successfully

I am a front-end newcomer😂
Thank you for your reading, have a good day!

**Comment by matthewmueller** [comment]
Hey @Grey0520, right now it's not easily possible, but it's very important and will be trivial once https://github.com/livebud/bud/discussions/56 is implemented.

To do it now, you can pull in the `*http.Request` and `http.ResponseWriter` directly and read and write cookies https://github.com/livebud/bud/pull/185/files to implement persistence.
